drivers/soundwire/generic_bandwidth_allocation.c

```diff
@@ -240,6 +240,9 @@ static int sdw_get_group_count(struct sdw_bus *bus,
 		return -ENOMEM;
 
 	list_for_each_entry(m_rt, &bus->m_rt_list, bus_node) {
+		if (m_rt->stream->state == SDW_STREAM_DEPREPARED)
```

Comment:
2 typos in the commit message, zoombie->zombie & not longer -> no longer

Reply:
fixed